docs/data/base/getting-started/overview/overview.md

```diff
@@ -0,0 +1,23 @@
+# MUI Base - Overview
+
+<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks, ideally suited for implementing a custom design system.</p>
+
+## Introduction
+
+MUI Base is library of unstyled React UI components.
+It features the same robust engineering as Material UI, but without implementing Material Design.
+
+With MUI Base, developers can rapidly build UIs with custom design systems, without the need to override any default style settings.
+
+## Why use MUI Base?
+
+MUI Base is ideally suited for developers working with a custom design system.
+
+The library includes prebuilt, unstyled components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
```

Comment:
I believe we should give a mention to accessibility support out-of-the-box and add something like "for you to own styling completely" about customizability—developers can use Base with whatever CSS solution out there.